src/components/useraccount/orderdetail/MainDetails.vue

```diff
@@ -0,0 +1,155 @@
+<template>
+  <div v-if="me">
+    <div class="row">
+      <div class="col-sm-6">
+        <div class="my-orders-order-dark-box">
+          <div class="my-orders-order-dark-box-title">{{ $t('shippingAddress') }}</div>
+          <div>
+            {{me.order.shippingAddress.title}}
+            {{me.order.shippingAddress.firstName}}
+            {{me.order.shippingAddress.lastName}}
+          </div>
+          <div>{{me.order.shippingAddress.streetName}} {{me.order.shippingAddress.streetNumber}}</div>
+          <div>{{me.order.shippingAddress.additionalStreetInfo}}</div>
+          <div>{{me.order.shippingAddress.city}}</div>
+          <div>{{me.order.shippingAddress.postalCode}} {{me.order.shippingAddress.region}}</div>
+          <div>{{me.order.shippingAddress.country}}</div>
+          <br>
+          <div>{{me.order.shippingAddress.contactInfo.phone}}</div>
+          <div>{{me.order.shippingAddress.contactInfo.email}}</div>
+        </div>
+      </div>
+      <div class="col-sm-6">
+        <div class="my-orders-order-dark-box">
+          <div class="my-orders-order-dark-box-title">{{ $t('billingAddress') }}</div>
+          <div>
+            {{me.order.billingAddress.title}}
+            {{me.order.billingAddress.firstName}}
+            {{me.order.billingAddress.lastName}}
+          </div>
+          <div>{{me.order.billingAddress.streetName}} {{me.order.billingAddress.streetNumber}}</div>
+          <div>{{me.order.billingAddress.additionalStreetInfo}}</div>
+          <div>{{me.order.billingAddress.city}}</div>
+          <div>{{me.order.billingAddress.postalCode}} {{me.order.billingAddress.region}}</div>
+          <div>{{me.order.billingAddress.country}}</div>
+          <br>
+          <div>{{me.order.billingAddress.contactInfo.phone}}</div>
+          <div>{{me.order.billingAddress.contactInfo.email}}</div>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-sm-6">
+        <div class="my-orders-order-dark-box">
+          <div class="my-orders-order-dark-box-title">{{ $t('shippingMethod') }}</div>
+          <div>
+            {{me.order.shippingInfo.shippingMethod.name}}
```

Comment:
Shipping info is also an optional field

Reply:
Also, have you checked the field `shippingMethodName`? I guess it's a faster alias to `shippingMethod.name`, but I'm not sure.

Reply:
well it would make sense if I was only fetching the name, but I also fetch description so I need to get the 'shippingMethod' object anyway